### PR TITLE
[refactor] bump packages version (angular, firebase, etc)

### PR DESCRIPTION
**Pull request by patrickgdl**
Let's revive this project? I have a lot of new ideas and a new layout is almost done. Check it [here](https://www.figma.com/file/qTJGVQprFSYcbCZHe5BRCL/Dashboard-Akademic?node-id=0%3A1).

The first thing for it is updating this project, the following things were updated:

- [x] @angular/* - from v6 to v12
- [x] angularfire2 to @angular/fire v6
- [x] primeng from v6 to v12 (I hope we someway drop this package, it's huge and we have a lot of possibilities, let's talk about it)
- [x] bootstrap from v3 to v5 (this one requires attention, because it broke **everything**, but it's ok because can we study now different approachs, like [Tailwind](https://tailwindcss.com/), maybe? Or pure SASS/CSS. Or even @angular/material or any other lib ui. Bootstrap today is so boring...
- [ ] I removed strict types and strict template types but we should enable it and fix any TS errors ASAP.

Updating Angular and Firebase there is a lot of breaking changes, you can see for yourself on the last 10 commits.

Hope you get excited as I am and let's f*cking do it!

### CODE REVIEW

File: src/app/app.routing.ts [hunk 1]
```diff
@@ -18,28 +18,54 @@ import { StudentGuard } from './guards/student.guard';
...
   { path: 'profile', component: ProfileComponent, canActivate: [AuthGuard] },
-  { path: 'student', loadChildren: './features/student/student.module#StudentModule', canActivate: [AuthGuard, StudentGuard] },
-  { path: 'admin', loadChildren: './features/admin/admin.module#AdminModule', canActivate: [AuthGuard, AdminGuard] },
-  { path: 'institution', loadChildren: './features/institution/institution.module#InstitutionModule', canActivate: [AuthGuard, InstitutionGuard] },
-  { path: 'controller', loadChildren: './features/controller/controller.module#ControllerModule', canActivate: [AuthGuard, ControllerGuard] },
-  { path: 'events', loadChildren: './features/events/events.module#EventsModule', canActivate: [AuthGuard] },
-  { path: '**', component: Error404Component }
+  {
+    path: 'student',
+    loadChildren: './features/student/student.module#StudentModule',
```
Comment (luizhc) [inline src/app/app.routing.ts]: new lazy loading missing

**Comment by luizhc** [comment]
Reviewed pull request. ✅

Just a simple line of code need to be adjust, otherwise it's perfect @patrickgdl 🎉

I'm very excited for it, let's dude? 🚀

**Review by luizhc — CHANGES_REQUESTED** [review]

File: src/app/app.routing.ts (hunk 1 above)
Comment (patrickgdl) [reply src/app/app.routing.ts]: Fixed.

**Review by luizhc — APPROVED** [review]
Perfect, thank you! 🚀

**Pull request merged**